docs/src/main/sphinx/connector/scylladb.rst

```diff
@@ -0,0 +1,38 @@
+==================
+ScyllaDB connector
+==================
+
+The ScyllaDB connector allows querying data stored in
+`ScyllaDB <https://www.scylladb.com//>`_.
+
+Requirements
+------------
+
+To connect to ScyllaDB, you need:
+
+* ScyllaDB version 3.0.0 or higher.
+* Network access from the Trino coordinator and workers to ScyllaDB.
+  Port 9042 is the default port.
+
+Configuration
+-------------
+
+To configure the ScyllaDB connector, create a catalog properties file
+``etc/catalog/scylladb.properties`` with the following contents,
+replacing ``host1,host2`` with a comma-separated list of the ScyllaDB
+nodes, used to discovery the cluster topology:
+
+.. code-block:: text
+
+    connector.name=scylladb
+    cassandra.contact-points=host1,host2
```

Comment:
My concern regarding this connector is that the `cassandra.xxxx` properties may be misleading while configuring it.
Also if the underlying driver evolves (see cassandra driver v3 -> v4 https://github.com/trinodb/trino/pull/7828 ) the settings will probably diverge.

Reply:
When we upgrade driver in Cassandra connector, ScyllaDB connector will be updated as well.